surveys/2023-annual-survey/questions.md

```diff
@@ -594,6 +594,21 @@ Aspects:
 >
 > This question gathers data on the communities perceptions of certain aspects of Rust at this point in time.
 
+### How do you build your Rust projects?
+
+Type: select one (optional)
+
+Aspects:
+
+- I use Cargo
+- I use some other build system
+- I combine Cargo and another build system
+- If you combine Cargo with (or just use) other build systems, which ones? (open response)
```

Comment:
Yes, the last option is not super clear. But I don't have the expertise to suggest examples. f.e. I don't have the context to appreciate the difference between your option 3 and 4.

```suggestion
- I just use Cargo
- I use some another build system (please specify) (open response)
- I combine Cargo and another build system (please specify) (open response)
```

:thinking: would it make it a bit clearer?

Reply:
If we can have two open responses, then this seems nice! :)

Reply:
> If we can have two open responses, then this seems nice! :)

It seems we cannot (tested on a SurveyHero account test account)